roles/edpm_telemetry/tasks/configure.yml

```diff
@@ -112,11 +112,14 @@
   when:
     - tls_crt_stat.stat.exists and tls_key_stat.stat.exists
   block:
-    - name: Create config file for node_exporter
-      ansible.builtin.template:
-        dest: "{{ edpm_telemetry_config_dest }}/node_exporter.yaml"
-        mode: "0644"
-        src: node_exporter.yaml.j2
+    - name: Create config file for exporters
+      ansible.builtin.include_tasks:
+        file: exporter_tls.yml
+        loop:
+          - node_exporter
+          - podman_exporter
+        loop_control:
```

Comment:
These are indented incorrectly. They need to be at the indentation level of the `ansible.builtin.include_tasks:`

Note the error you're getting during the telemetry Ansible deployment:
https://softwarefactory-project.io/zuul/t/rdoproject.org/build/e6da61144cfe4525a91242d82b3563fd/log/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/telemetry-edpm-deployment-ipam-openstack-edpm-ipam-zdrn5/logs/telemetry-edpm-deployment-ipam-openstack-edpm-ipam.log

Reply:
Indeed, thanks for the pointer.